source/dotnet/Library/AdaptiveCards/AdaptiveCardConverter.cs

```diff
@@ -44,5 +48,30 @@ public override bool CanConvert(Type objectType)
         {
             return typeof(AdaptiveCard).GetTypeInfo().IsAssignableFrom(objectType.GetTypeInfo());
         }
+
+        private AdaptiveCard MakeFallbackTextCard(JObject jObject)
+        {
+            // Retrieve fallbackText and language defined by parsed json
+            string language = jObject.Value<string>("language");
+            string fallbackText = jObject.Value<string>("fallbackText");
+
+            if (fallbackText == null || fallbackText == "")
```

Comment:
Should be using `String.IsNullOrEmpty()` here. ([docs](https://docs.microsoft.com/en-us/dotnet/api/system.string.isnullorempty?view=netframework-4.7.2#System_String_IsNullOrEmpty_System_String_))